pkg/sdn/api/types.go

```diff
@@ -60,3 +60,42 @@ type NetNamespaceList struct {
 	unversioned.ListMeta
 	Items []NetNamespace
 }
+
+// EgressNetworkPolicyRuleType gives the type of an EgressNetworkPolicyRule
+type EgressNetworkPolicyRuleType string
+
+const (
+	EgressNetworkPolicyRuleAllow EgressNetworkPolicyRuleType = "Allow"
+	EgressNetworkPolicyRuleDeny  EgressNetworkPolicyRuleType = "Deny"
+)
+
+// EgressNetworkPolicyPeer specifies a target to apply egress policy to
+type EgressNetworkPolicyPeer struct {
+	CIDRSelector string
+}
+
+// EgressNetworkPolicyRule contains a single egress network policy rule
+type EgressNetworkPolicyRule struct {
+	Type EgressNetworkPolicyRuleType
+	To   EgressNetworkPolicyPeer
+}
+
+// EgressNetworkPolicySpec provides a list of policies on outgoing traffic
+type EgressNetworkPolicySpec struct {
+	Egress []EgressNetworkPolicyRule
+}
+
+// EgressNetworkPolicy describes the current egress network policy
```

Comment:
Flush out this godoc - preferably 3-4 sentences.  End user needs to be able to figure out what this object does and how it is used from the swagger doc.